### PR TITLE
Free category with terminal object

### PR DESCRIPTION
**Pull request by hejohns**

### CODE REVIEW

**Comment by hejohns** [comment]
rebased onto #81

**Comment by maxsnew** [comment]
@hejohns is this still a draft or is it ready to review?

**Comment by hejohns** [comment]
What’s there is ready for review.
I wasn’t sure if we wanted anything else in the same PR, but I can just put
in another PR.

File: Cubical/Categories/Constructions/Free/CategoryWithTerminal.agda [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  variable
+    ℓg ℓg' ℓC ℓC' ℓCᴰ ℓCᴰ' : Level
+
+CategoryWithTerminal' : (ℓC ℓC' : Level) → Type _
+CategoryWithTerminal' ℓC ℓC' = Σ[ C ∈ Category ℓC ℓC' ] Terminal' C
+
+-- freely throw in a terminal object
+module _ (Ob : Type ℓg) where
+
+    -- adjoin a new terminal object
```
Comment (maxsnew) [inline Cubical/Categories/Constructions/Free/CategoryWithTerminal.agda:29]: please use 2 spaces for indentation, not 4.

**Review by maxsnew — CHANGES_REQUESTED** [review]
Looks good just change the indentation issue

File: Cubical/Categories/Constructions/Free/CategoryWithTerminal.agda (hunk 1 above)
Comment (hejohns) [reply Cubical/Categories/Constructions/Free/CategoryWithTerminal.agda]: patched with https://github.com/maxsnew/cubical-categorical-logic/pull/79/commits/5f221ae2653f7408a1ff184dc7efc4f3fc02c924

**Review by maxsnew — APPROVED** [review]

**Pull request merged**